falcosidekick/CHANGELOG.md

```diff
@@ -5,6 +5,28 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 1.0.0
```

Comment:
I wasn't clear, sorry.

For this chart, ok for bumping to 0.3.0. 

But, we also need to bump up the version of `falcosidekick-ui` to `v1.0.0`: https://github.com/falcosecurity/charts/blob/b52c1914b4731fc65fe5c5d515d712aa303c8d08/falcosidekick/values.yaml#L248

and you can increase the retention to 200 too: https://github.com/falcosecurity/charts/blob/b52c1914b4731fc65fe5c5d515d712aa303c8d08/falcosidekick/values.yaml#L255

Reply:
no worries, it is my mistake, I couldn't understand it correctly 🤦

Reply:
Okay, it is done now.